pkgs/development/tools/drm_info/default.nix

```diff
@@ -1,25 +1,25 @@
 { stdenv, fetchFromGitHub
-, libdrm, json_c
+, libdrm, json_c, pciutils
 , meson, ninja, pkgconfig
 }:
 
 stdenv.mkDerivation rec {
   pname = "drm_info";
-  version = "2.1.0";
+  version = "2.2.0";
 
   src = fetchFromGitHub {
     owner = "ascent12";
     repo = "drm_info";
     rev = "v${version}";
-    sha256 = "1i5bzkgqxjjw34jpj1x1gfdl3sz0sl6i7s787a6mjjslsc5g422l";
+    sha256 = "0s4zp8xz21zcpinbcwdvg48rf0xr7rs0dqri28q093vfmllsk36f";
   };
 
   nativeBuildInputs = [ meson ninja pkgconfig ];
-  buildInputs = [ libdrm json_c ];
+  buildInputs = [ libdrm json_c pciutils ];
 
   meta = with stdenv.lib; {
     description = "Small utility to dump info about DRM devices.";
-    homepage = "https://github.com/ascent12/drm_info";
```

Comment:
it is good to have the link there to be able to copy it, for example when you review a pr

Reply:
I agree, but the general move on derivations fetched from github seems to be to move to the inherit, to lessen duplicate information.

Reply:
I don't think duplicate information is a problem here. Especially not when it makes it so I have to type a URL rather than copy-paste (or drag'n'drop) one.

Reply:
I don't have a strong opinion on this and reinstated the old URL.